HW4_Trofimov/das_protein_tools.py

```diff
@@ -0,0 +1,361 @@
+# importing necessary modules
+import protein_dict as pd
+from random import choice
+
+
+AMINO_LETTERS = [
```

Comment:
Не очень вижу смысл в списке (см. ниже комментарий по поводу использования этой переменной, там совет использовать множества)
Используется эта переменная один раз, и то там лучше применять множества. 
Преобразовать список в множество занимает какое-то количество времени, поэтому эту переменную сразу надо определять как множество, и ниже использовать уже как есть (я ниже показала вариант кода, где из списка делается множество, но эти два комментария связаны, так что учтите)

```suggestion
AMINO_LETTERS = set("ACDEFGHIKLMNPQRSTVWY")
```